### PR TITLE
Add support for suffix destination filtering.

### PR DESCRIPTION
**Pull request by jwcarman**

### CODE REVIEW

File: activemq-client/src/main/java/org/apache/activemq/filter/SuffixDestinationFilter.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+
+/**
+ * Matches messages which match a suffix like "<.A.B"
+ *
+ *
+ */
+public class SuffixDestinationFilter extends DestinationFilter {
+
+    private String[] suffixes;
```
Comment (peteryhwong) [inline activemq-client/src/main/java/org/apache/activemq/filter/SuffixDestinationFilter.java]: Can these not be final?

Comment (jwcarman) [inline activemq-client/src/main/java/org/apache/activemq/filter/SuffixDestinationFilter.java]: Yes. I was mimicking the existing code I believe

**Comment by hzbarcea** [comment]
Wow, 5.11... @jwcarman do you still want this patch applied? If so I'll try to test and port as necessary. Please say no :).

**Comment by jwcarman** [comment]
@hzbarcea well, if it hasn't been merged by now (2.5 years later), I don't guess I really need it at all. :)

**Comment by hzbarcea** [comment]
@jwcarman fair enough. If you want us to be more responsive next time ping me. You know how. Happy Holidays!

**Pull request closed without merging**